Dockerfile.deb-stable

```diff
@@ -0,0 +1,14 @@
+FROM debian:stable
```

Comment:
jessie uses libc 2.19, while the prior stable wheezy uses 2.13 which may be a better target